src/v2/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -175,6 +177,9 @@ export const OtherWorksFragmentContainer = createFragmentContainer<{
       context {
         __typename
       }
+      seriesArtist: artist(shallow: true) {
```

Comment:
We only need the artist id to get the artist series, so `shallow: true` will avoid fetching the artist (the id is already included in the nested artist JSON as part of an artwork response).